analyzers/backend/docker/container.go

```diff
@@ -65,7 +66,7 @@ func (d *DockerClient) StartDockerContainer() error {
 	// Prepare the network config
 	networkConfig := network.NetworkingConfig{}
 	platform := v1.Platform{
-		Architecture: "amd64",
+		Architecture: runtime.GOARCH,
```

Comment:
What if the user specifies the platform using `--platform` flag? 😅

Reply:
Resolved in current commit.